res/qml/Mixxx/qmldir

```diff
@@ -0,0 +1,8 @@
+module Mixxx
+linktarget mixxx-libplugin
+optional plugin mixxx-libplugin
+classname MixxxPlugin
+typeinfo mixxx-lib.qmltypes
```

Comment:
This is required to keep qmllint happy

```suggestion
typeinfo mixxx-lib.qmltypes
import QtQuick
```

Reply:
This is unrelated and should be taken care of in a separate PR, that also adds this import to `qt_add_qml_module`. The `qmldir` is the same one that Qt generates automatically, just with the `MathUtils` line added.

Reply:
I have already taken care in https://github.com/mixxxdj/mixxx/pull/4679. So If we merge that first, we should add the import here as well.